pagopa/io-app
# chore: [IOBP-481] Remove DS feature flag to show new barcode scan screen

## Short description
This PR removes the feature flag navigation to the new barcode screen

## List of changes proposed in this pull request
- Removed the check on the DS feature flag while the user taps on the `Paga un avviso` CTA, navigating by default to the new barcode scan screen

## How to test
- Open the wallet home page
- Tap on `Paga un avviso` CTA
- You should be able to navigate to the new barcode scan screen even with the DS flag deactivated
